### DISCUSSION TITLE
When 26.0.8 version will be released? (I have check milestones but not found timeline there)

### DISCUSSION THREAD

**Discussion by bmk9727** [keycloak/keycloak#36144] [Q&A]
In our org we are using keycloak as centralized IAM server and we had started it with keycloak 25.0.1 version and to stay updated with latest release we were planning to upgrade it to keycloak:26.0.7 but found out that one of core functionality for admin event representation that we were using as audit perspective is breaking in 26.0.7(https://github.com/keycloak/keycloak/issues/35483) So we are bit curious if 26.0.8 will be out soon with this fix or not?
I had check this milestone page: https://github.com/keycloak/keycloak/milestones but was not able to find 26.0.8 version's timeline.

**Comment by Tim-Schwalbe** [comment]
I am also in need for this as the mySQL migration is failing with versions <26.0.8

**Comment by jonkoops** [comment]
Probably early next year due to the low availability of the team during the Christmas/December break.

**Reply by bmk9727** [reply]
Got it, Thanks @jonkoops
